syntax = "proto3";

package api.chat.request;

message GetInfo {
    uint64 chat_id = 1;
}
